ILSpy/App.xaml.cs

```diff
@@ -59,6 +59,10 @@ public partial class App : Application
 
 		public static IExportProvider ExportProvider { get; private set; }
 
+		private readonly SettingsService settingsService;
+		
+		public static SettingsService SettingsService => Current.settingsService;
```

Comment:
I am not a fan of new static state... @tom-englert what would be the current WPF best practice to solve this problem? Given that you have refactored pretty much everything UI and DI-related in ILSpy to follow best practices. Thank you very much!

Reply:
Yes, settings must only be used via DI.
@Hertzole, @siegfriedpammer a fixed version is here: https://github.com/icsharpcode/ILSpy/pull/3405. Either adapt this PR to match #3405 and abandon #3405, or abandon this and use #3405, whatever you prefer.

Reply:
@tom-englert thank you very much for your fast response and taking the time to look into this.